src/Compilers/CSharp/Test/Emit/PDB/PDBTests.cs

```diff
@@ -492,9 +492,9 @@ public void CustomDebugEntryPoint_DLL()
   </files>
   <entryPoint declaringType=""C"" methodName=""F"" />
   <methods/>
-</symbols>", debugEntryPoint: f, options: PdbValidationOptions.ExcludeScopes | PdbValidationOptions.ExcludeSequencePoints | PdbValidationOptions.ExcludeCustomDebugInformation);
+</symbols>", debugEntryPoint: f.GetPublicSymbol(), options: PdbValidationOptions.ExcludeScopes | PdbValidationOptions.ExcludeSequencePoints | PdbValidationOptions.ExcludeCustomDebugInformation);
```

Comment:
>.GetPublicSymbol() [](start = 31, length = 18)

Call to `GetPublicSymbol()` is repeated a lot. Would it make sense for `GetMember<T>` helper to return the public symbol directly, or do we have many cases that need the internal symbol? If the latter, then I'd prefer calling `GetPublicSymbol()` in the declaration of `f`.

Reply:
> Call to GetPublicSymbol() is repeated a lot. Would it make sense for GetMember<T> helper to return the public symbol directly, or do we have many cases that need the internal symbol? If the latter, then I'd prefer calling GetPublicSymbol() in the declaration of f to avoid repetition.

I don't think we want ```GetMember<T>``` helper to return the public symbol directly when called on internal symbol. Also, in this file there are only two call-sites of ```VerifyPdb``` that use the helper. 


---
In reply to: [340385499](https://github.com/dotnet/roslyn/pull/39498#discussion_r340385499) [](ancestors = 340385499)